include/libint2/engine.impl.h

```diff
@@ -1687,6 +1696,9 @@ __libint2_engine_inline const Engine::target_ptr_vec& Engine::compute2(
 
       case BraKet::xx_xs: assert(false && "this braket is not supported"); break;
       case BraKet::xs_xx: {
+        // TODO, now that we have arbitrary-order deriv maps, this logic is no longer needed, right?
```

Comment:
It seems (per https://github.com/evaleev/libint/blob/master/include/libint2/engine.impl.h#L600):
```
const int ket_lmax = (hard_default_lmax_ == std::numeric_limits<int>::max()) ? hard_lmax_ : hard_default_lmax_;
```

Reply:
Done